themes/survey/fruity_twentythree/views/layout_survey_list.twig

```diff
@@ -112,18 +112,18 @@ NOTE: Feel free to ask for functions, variables, etc, that you feel are missing
 
             {# Replace the content block #}
             {% block content %}
-                <div class="container-md">
+                <div class="container-md" role="none">
                     {{ registerTemplateCssFile('./css/survey-list.css') }}
 
-                    <div class="{{ aSurveyInfo.class.surveylistrow }} row"
+                    <main class="{{ aSurveyInfo.class.surveylistrow }} row"
```

Comment:
Need role="main"
https://www.w3.org/Translations/WCAG21-fr/#name-role-value

To get RGAA in France: you need both `<main  role="main">`